modulus/datapipes/healpix/timeseries_dataset.py

```diff
@@ -189,24 +189,35 @@ def __init__(
 
         self.input_scaling = None
         self.target_scaling = None
+        self.constant_scaling = None
+        self.constants = None
         if self.scaling:
             self._get_scaling_da()
 
+        # setup constants
+        if "constants" in self.ds.data_vars:
+            # extract from ds:
+            const = self.ds.constants.values
+
+            if self.scaling:
+                const = (const - self.constant_scaling["mean"]) / self.constant_scaling[
+                    "std"
+                ]
+
+            # transpose to match new format:
+            # [C, F, H, W] -> [F, C, H, W]
+            self.constants = np.transpose(const, axes=(1, 0, 2, 3))
+
+        self.get_constants()
```

Comment:
Why is this needed here?

Reply:
Bottom line isn't needed, it should have been deleted. Fixed.